### DISCUSSION TITLE
Question Regarding Nano Instance Availability for Pro Plan Users

### DISCUSSION THREAD

**Discussion by Wunhyeon** [supabase/supabase#29967] [Questions]
I am currently on the Pro plan and operating one project with a Micro instance. I would like to start an additional lightweight project using a Nano instance for free. However, when creating a new project, the minimum available option is a Micro instance, which incurs additional costs. Is it not possible for Pro plan users to use a Nano instance?

**Comment by silentworks** [comment]
In order to do this you should create another organization in your account and create the free project in that account which will have a Nano instance. Nano instances aren't available on the Pro plan I think. You get 2 free projects per account, but the free projects cannot be in the paid organization.
